docs/Settings.md

```diff
@@ -2912,23 +2912,73 @@ P gain of Heading Hold controller (Fixedwing)
 
 ---
 
-### nav_fw_land_dive_angle
```

Comment:
Removing a parameter will break backwards compatibility. This should probably wait until the next major release.

Reply:
I'm actually confused why this was removed. The legacy landing mode should be still there if no landing location is defined?

Reply:
True. It at least needs to be there as a fallback. For if no safehomes with landing strips were detected when arming.